sdk/tools/azure-openrewrite/src/main/resources/META-INF/rewrite/migrationRecipes/azure-storage-blob-migration.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------
+# Migration Recipes for Azure-Storage-Blob Library
+# --------------------------------------------------------------------
+#
+type: specs.openrewrite.org/v1beta/recipe
+name: com.azure.openrewrite.recipes.migrateAzureStorageBlob
+displayName: Migrate from azure-storage-blob to next generation stack
+description: This recipe migrates the Azure Storage Blob library to the next generation stack.
+recipeList:
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: com.azure.storage.blob.BlobClient
```

Comment:
Shouldn't these transitions be coming from the generation itself, why do we need these?

Reply:
@samvaity Correct, this was just for proof-of-product. Storage hasn't actually begun their migration yet so none of these recipes are valid.